src/client/src/combineReducers.ts

```diff
@@ -3,7 +3,7 @@ import compositeStatusServiceReducer from './compositeStatusServiceOperations'
 import connectionStatusReducer from './connectionStatusOperations'
 import { currencyPairReducer } from './currencyPairsOperations'
 import { pricingServiceReducer } from './pricingOperations'
-import { analyticsReducer } from './ui/analytics/index'
+import { analyticsReducer } from './ui/Analytics'
```

Comment:
have you renames the directory name to uppercase? we should keep the directory names in lowercase